constantinCurtis/src/main/java/data/collection/task/classes/AbstractOrder.java

```diff
@@ -0,0 +1,104 @@
+package data.collection.task.classes;
+
+import data.collection.task.Interfaces.Order;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public abstract class AbstractOrder implements Order {
+    private final String id;
+    private final LocalDateTime date;
+    private List<Item> items;
```

Comment:
Please change to protected in order to proper inherit those variables.

Reply:
Please read my teams message